src/NuGet.Protocol.Core.v3/RemoteRepositories/STSAuthHelper.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+#if !DNXCORE50
+using System.IdentityModel.Protocols.WSTrust;
+using System.IdentityModel.Tokens;
+using System.Linq;
+#endif
+using System.Net;
+using System.Net.Http;
+#if !DNXCORE50
+using System.ServiceModel;
+using System.ServiceModel.Security;
+#endif
+using System.Text;
+using NuGet.Configuration;
+
+namespace NuGet.Protocol.Core.v3
+{
+    public class STSAuthHelper
```

Comment:
Looks similar to http://nuget.codeplex.com/SourceControl/latest#src/Core/Http/STSAuthHelper.cs, but not the same. Why?!